source/_static/scss/includes/_misc.scss

```diff
@@ -379,4 +379,16 @@ figcaption {
 table thead tr.header {
    background-color: lightgray;
    box-shadow: none;
+}
+
+// --------------------------------------
+// Improving readability of ordered lists
+// --------------------------------------
+
+div.procedure ol.arabic li > p:first-child {
+   font-weight: bold;
+}
+
+div.procedure ol.arabic li ul.simple li p {
```

Comment:
I feel dumb having done this. @rushenn @kaankabalak can we make this cleaner? for some reason it cascades down w/o this.

Reply:
Sure @ravindk89, I can take a look